### ISSUE TITLE
Few hopefully simple questions

### ISSUE THREAD

**Issue by LiamKenyon** [Neo-1977/N0DE-DL#2]
Just been taking a look at this as I’m fairly new to nodejs and saw you mention on a wvtohls gist.

I’ve managed to get things installed etc on u20.04 and get the app to run but when ever browsing any of the routes I get “Cannot GET /“.

As said new to nodejs so probably doing something wrong.

I have noticed that it’s referencing a config file (env file) which isn’t included which is probably causing issues but I’ve not tried to recreate that yet unless you have a blank one on hand?

Any pointers? As said new to nodejs.

**Comment by Neo-1977** [comment]
everything required is there which will work with redis and pm2

**Comment by LiamKenyon** [comment]
No problem. As said got most sorted just a few gripes. Will go through all the files and pull out what’s being reference for a env file. 

Joys of learning a new language.

**Comment by DRM-Scripts** [comment]
From memory you'll need to make you have the correct files in binaries/linux I remember one was a bit of a pain to get hold off.

Edit: Quick look at the code and think it was the foobar/decrypter binarie.

**Comment by LiamKenyon** [comment]
> From memory you'll need to make you have the correct files in binaries/linux I remember one was a bit of a pain to get hold off.
> 
> Edit: Quick look at the code and think it was the foobar/decrypter binarie.

Yeah that's also where I got stuck, the foobar let in video-manager stopped me and haven't had time to look any further.

If you or @Neo-1977 can point me in the right direction for that binary that would be appreciated.

**Comment by Neo-1977** [comment]
I wont be supplying that thanks.

**Comment by Neo-1977** [comment]
Public decompiled versions of the Binary is on this site. All i did was make it into a binary.
